x-pack/plugins/cases/public/components/recent_cases/no_cases/index.test.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+
+import { TestProviders } from '../../../common/mock';
+import { NoCases } from '.';
+
+describe('RecentCases', () => {
+  it('if no cases, you should be able to create a case by clicking on the link "start a new case"', () => {
```

Comment:
nit: I think this is testing if the correct href is there and does not simulate a user click on the link.